src/main/java/com/example/nlern/Constants.java

```diff
@@ -0,0 +1,5 @@
+package com.example.nlern;
+
+public class Constants {
+    public static final int NUMBER_OF_QUESTIONS = 10;
```

Comment:
Do you really need a whole new class for just a single constant?

Reply:
this is for the future. there will be many of them

Reply:
This is not a justification for creating a new class. For now, refrain from doing so and create constants where you intent to use them.

Reply:
rewritten